service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -203,8 +205,13 @@ public ControlledGcsBucketResource updateGcsBucket(
             .newJob()
             .description(jobDescription)
             .flightClass(UpdateControlledGcsBucketResourceFlight.class)
-            .request(resource)
+            .resource(resource)
             .userRequest(userRequest)
+            .operationType(OperationType.UPDATE)
+            .workspaceId(resource.getWorkspaceId().toString())
+            // TODO: conflict with resource name
```

Comment:
Please add ticket to TODO and specify the action for the TODO.

Reply:
Done - added PF-1282